rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -275,6 +277,27 @@ public Repository getRepository() {
         return repository;
     }
 
+    public BlockExecutorFactory getBlockExecutorFactory() {
+        if (blockExecutorFactory == null) {
+            blockExecutorFactory = buildBlockExecutorFactory();
+        }
+
+        return blockExecutorFactory;
+    }
+
+    private BlockExecutorFactory buildBlockExecutorFactory() {
```

Comment:
the `buildXXX`/`getXXX` separation is not needed when no one is overriding this object creation

Reply:
Done